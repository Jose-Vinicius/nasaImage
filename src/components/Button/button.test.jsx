import { render, screen } from '@testing-library/react';
import {expect, it, test} from 'vitest';
import { Button } from '.';

describe('Botão utilizado na aplicação',() => {
    it('Botão com texto correto', () => {
        render(
            <Button>Buscar Imagem</Button>
        )

        const textButton = 'Buscar Imagem'
        const button = screen.getByRole('button', {name: textButton})
        
        expect(button).toMatchSnapshot();
       
        
    })
})
