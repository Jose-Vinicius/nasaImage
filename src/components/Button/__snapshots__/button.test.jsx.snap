// Vitest Snapshot v1

exports[`Botão utilizado na aplicação > Botão com texto correto 1`] = `
<button>
  Buscar Imagem
</button>
`;
